tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/manifest.ini

```diff
@@ -76,10 +76,12 @@ skip-if = device == "desktop"
 dsds = true
 # Bug 1168072 - Intermittent failures in Jenkins in test_dialer_dsds_make_phone_call_with_default_sim.py
 stable = false
-skip-if = device == "desktop"
```

Comment:
I started to clean up, like explained in bug 1189747. I'm fine to not do it now and wait for bug 1189747.